scripts/docker-cross-tag.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024-2024  The DOSBox Staging Team
+
+# This script requires the following programs to be installed as prerequisites:
+#   - curl
+#   - jq
+#   - sha256sum
+
+usage() {
+    printf "%s\n" "\
+Usage: $0 FUNC OPTIONS
+
+Utility script to assist with docker image tags
+
+FUNC must be one of:
+  tag-exists         Test if image tag exists in the GitHub Container Registry (GHCR).
+                     Takes two options: IMAGE_NAME, TAG.
```

Comment:
One last thing, maybe remove the comma from after IMAGE_NAME? I find it a bit misleading.